plugins/redfish/fu-redfish-common.h

```diff
@@ -9,14 +9,16 @@
 #include <gio/gio.h>
 
 /* SMBIOS */
-#define REDFISH_SMBIOS_TABLE_TYPE			0x42
+#define REDFISH_SMBIOS_TABLE_TYPE			0x2a	/* 42 */
```

Comment:
this a mistake?  should split to separate commit i'd say

Reply:
Deliberate, unfortunately. a "type 42 table" is 42 decimal. I'll split it out and also backport that line.